source/extensions/listener_managers/listener_manager/lds_api.cc

```diff
@@ -68,20 +68,32 @@ LdsApiImpl::onConfigUpdate(const std::vector<Config::DecodedResourceRef>& added_
   ListenerManager::FailureStates failure_state;
   absl::node_hash_set<std::string> listener_names;
   std::string message;
+
   for (const auto& resource : added_resources) {
     envoy::config::listener::v3::Listener listener;
+
+    auto onError = [&](std::string error_message) {
+      failure_state.push_back(std::make_unique<envoy::admin::v3::UpdateFailureState>());
+      auto& state = failure_state.back();
+      state->set_details(message);
```

Comment:
I think this needs to be:
```suggestion
      state->set_details(error_message);
```